torch/_inductor/mkldnn_ir.py

```diff
@@ -799,22 +803,39 @@ def __init__(
     def codegen(self, wrapper):
         # Parser the inputs and constant
         args = [x.codegen_reference() for x in self.inputs]
-        const_args = []
-        const_args.extend(self.codegen_const_args())
+        const_arg_names = [
+            "act_scale",
```

Comment:
Do we need to force the exact same name of formal parameters?
There are some mis-using of `act_scale` and `x_scale` between here and op registration, will it be an issue? https://github.com/pytorch/pytorch/blob/56f2917bef45c33d7b29b3247dcf751a898186e5/aten/src/ATen/native/quantized/library.cpp#L266

Reply:
It needs to be an exact match. Let me update it.

Ideally we want to declare all Tensor args before scalar ops. We can't change these ops for BC reason, but let's follow the convention for future ops.